stdlib/public/core/BridgeObjectiveC.swift

```diff
@@ -415,26 +408,32 @@ public struct AutoreleasingUnsafeMutablePointer<Pointee /* TODO : class */>
   public subscript(i: Int) -> Pointee {
     @_transparent
     get {
-      _debugPrecondition(!_isNull)
       // We can do a strong load normally.
       return (UnsafePointer<Pointee>(self) + i).pointee
     }
   }
 
-  /// Create an instance initialized with `nil`.
-  @_transparent
-  public init(nilLiteral: ()) {
-    self._rawValue = _nilRawPointer
+  /// Explicit construction from an UnsafeMutablePointer.
+  ///
+  /// This is inherently unsafe; UnsafeMutablePointer assumes the
+  /// referenced memory has +1 strong ownership semantics, whereas
```

Comment:
Maybe that just falls out of the language rules, but it's worth calling out in *some* form. (I didn't write this text; it's copied from the initializer above.)